frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Kebab, ResourceKebab } from '@console/internal/components/utils';
+import { sortable } from '@patternfly/react-table';
+import { referenceForModel, VolumeSnapshotClassKind } from '@console/internal/module/k8s';
+import { TableRow, TableData, Table, ListPage } from '@console/internal/components/factory';
+import { VolumeSnapshotClassModel } from '@console/internal/models';
+
+const tableColumnClasses = [
+  '', // name
+  classNames('pf-m-hidden', 'pf-m-visible-on-md'), // Driver
+  classNames('pf-m-hidden', 'pf-m-visible-on-md'), // Deletion Policy
+  Kebab.columnClass,
+];
+
+const Header = () => [
+  {
+    title: 'Name',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[0] },
+  },
+  {
+    title: 'Driver',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[1] },
+  },
+  {
+    title: 'Deletion Policy',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[2] },
+  },
+  {
+    title: '',
+    props: { className: tableColumnClasses[3] },
+  },
+];
+
+type RowProps = {
+  obj: VolumeSnapshotClassKind;
+  index: number;
+  style: object;
+  key: string;
+};
+
+const Row: React.FC<RowProps> = ({ obj, index, style, key }) => {
+  const { name } = obj?.metadata || {};
```

Comment:
`obj` -> `volumeSnapshotClass` something like this

Reply:
Is it really required?

Reply:
its more understandable IMO.